command/events.go

```diff
@@ -147,19 +148,24 @@ func (c *EventsSubscribeCommands) subscribeRequest(client *api.Client, path stri
 			HTTPClient: client.CloneConfig().HttpClient,
 			HTTPHeader: client.Headers(),
 		})
-		if err != nil {
-			if resp != nil {
-				if resp.StatusCode == http.StatusNotFound {
-					return fmt.Errorf("events endpoint not found; check `vault read sys/experiments` to see if an events experiment is available but disabled")
-				} else if resp.StatusCode == http.StatusTemporaryRedirect {
-					url = resp.Header.Get("Location")
-					continue
-				}
-			}
+
+		if err == nil {
+			break
+		}
+
+		switch {
+		case resp == nil:
+			return err
+		case resp.StatusCode == http.StatusTemporaryRedirect:
```

Comment:
I realize now that it might be good to follow other `30x` redirects, in case the Vault endpoint is behind a proxy or load balancer.

Not necessarily needed to be fixed in this PR, but at least, reminding myself.